docs/src/main/sphinx/connector/kudu.rst

```diff
@@ -6,15 +6,17 @@ The Kudu connector allows querying, inserting and deleting data in `Apache Kudu`
 
 .. _Apache Kudu: https://kudu.apache.org/
 
-
 Compatibility
 -------------
```

Comment:
Is this section empty now? Should it be removed?

Reply:
I removed that section.